django_dbq/models.py

```diff
@@ -137,3 +137,17 @@ def run_creation_hook(self):
             logger.info("Running creation hook %s for new job", creation_hook_name)
             creation_hook_function = import_string(creation_hook_name)
             creation_hook_function(self)
+
+    @staticmethod
+    def get_queue_depths_dict():
```

Comment:
Thoughts on whether this needs the `dict` suffix? Putting the type in the method name feels weird
```suggestion
    def get_queue_depths():
```

Reply:
yeah that's a fair point! i'll update that